content/docs/gke/deploy/project-setup.md

```diff
@@ -25,13 +25,13 @@ Follow these steps to set up your GCP project:
 1. Go to the following pages on the GCP Console and ensure that the 
   specified APIs are enabled:
 
-  * [Compute Engine API](https://console.cloud.google.com/apis/library/compute.googleapis.com)
-  * [Kubernetes Engine API](https://console.cloud.google.com/apis/library/container.googleapis.com)
-  * [Identity and Access Management (IAM) API](https://console.cloud.google.com/apis/library/iam.googleapis.com)
-  * [Deployment Manager API](https://console.cloud.google.com/apis/library/deploymentmanager.googleapis.com)
-  * [Cloud Resource Manager API](https://console.developers.google.com/apis/library/cloudresourcemanager.googleapis.com)
-  * [Cloud Filestore API](https://console.developers.google.com/apis/library/file.googleapis.com)
-  * [AI Platform Training & Prediction API](https://console.developers.google.com/apis/library/ml.googleapis.com)
+    * [Compute Engine API](https://console.cloud.google.com/apis/library/compute.googleapis.com)
```

Comment:
Could you talk me through the reason for updating this file? The page on master seems good without the update: https://master.kubeflow.org/docs/gke/deploy/project-setup/